streamly.cabal

```diff
@@ -691,13 +689,11 @@ library
 
                      -- XXX to be removed
                      , transformers-base >= 0.4   && < 0.5
-                     , primitive         >= 0.5.4 && < 0.8
                      , heaps             >= 0.3     && < 0.5
 
                     -- concurrency
                      , atomic-primops    >= 0.8   && < 0.9
                      , lockfree-queue    >= 0.2.3 && < 0.3
-
```

Comment:
Do not remove this line, it groups the unicode dependency with concurrency dependencies.